hack/test.sh

```diff
@@ -53,5 +53,5 @@ if [[ -n "${GOFLAGS:-}" ]]; then
 fi
 
 echo go test "${TARGETS}" ${TIMEOUT} ${RUN_SINGLE_CASE} ${VERBOSE} ${DISABLE_CACHE}
-go test "${TARGETS}" ${TIMEOUT} ${RUN_SINGLE_CASE} ${VERBOSE} ${DISABLE_CACHE}
+go test -gcflags="-l" "${TARGETS}" ${TIMEOUT} ${RUN_SINGLE_CASE} ${VERBOSE} ${DISABLE_CACHE}
```

Comment:
Just curious why add `-gcflags="-l"` and what different behaviors can we observe before and after?

Reply:
gomonkey suggests it: 
https://github.com/bouk/monkey/blob/master/README.md
```
Monkey sometimes fails to patch a function if inlining is enabled. Try running your tests with inlining disabled, for example: go test -gcflags=-l. The same command line argument can also be used for build.
```

Reply:
Cool. Thanks for the info.